examples/plot_crcns_dataset_example.py

```diff
@@ -3,7 +3,8 @@
 CRCNS DataSet Example
 ============================================
 
-A demonstration of Spykes's functionality using a dataset from CRCNS
+A demonstration of Spykes's functionality to reproduce a figure from Li et al's
```

Comment:
hey @pavanramkumar did you or Hugo write the NeuroPop/CRCNS/Reaching Dataset examples?

Thanks!